symphony-bdk-core/src/main/java/com/symphony/bdk/core/config/model/BdkClientConfig.java

```diff
@@ -0,0 +1,33 @@
+package com.symphony.bdk.core.config.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
that could be configured globally in the mapper to avoid repeating it

Reply:
@symphony-hong +1